documentation/docs/developers/strategies/vwap.md

```diff
@@ -0,0 +1,53 @@
+# VWAP
+
+The VWAP strategy is a common algorithmic execution strategy which allows traders to account for slippage by splitting up orders over time. Specifically, the VWAP strategy helps traders minimize slippage when buying or selling large orders. The methods utilized in VWAP make the strategy more useful to traders and will help when creating future, more complex strategies.
```

Comment:
This is the old TWAP description... and it doesn't description VWAP well.

Change to this..

> The VWAP strategy is a common algorithm execution strategy which allows traders to execute large orders with controlled and limited market impact. It accomplishes this by measuring the market impact of each order before making new orders, and splitting up the trader's large order into smaller orders that have limited market impact. Unlike the TWAP strategy, VWAP doesn't wait for a specific interval before making new orders - it only waits for the last order to complete before making the next order.